src/patternfly/components/Nav/nav.scss

```diff
@@ -226,6 +229,11 @@
     font-weight: var(--pf-c-nav__list-link--FontWeight);
     color: var(--pf-c-nav__list-link--Color);
 
+    @media screen and (min-width: $pf-global--breakpoint--md) {
+      --pf-c-nav__list-link--PaddingRight: var(--pf-c-nav__list-link--PaddingRight--md);
```

Comment:
should the media query modifier in the var name come before the property? like `var(--pf-c-nav__list-link--md--PaddingRight);

Reply:
oh ya I forgot about that change.